src/main/java/stelnet/ui/Cargo.java

```diff
@@ -8,19 +8,11 @@
 
     private CargoAPI cargo;
     private String emptyDescription;
-    private float width;
-    private float height;
 
-    public Cargo(CargoAPI cargo, String emptyDescription, float width, float height) {
+    public Cargo(CargoAPI cargo, String emptyDescription, Size size) {
```

Comment:
I think it would be better to rename "emptyDescription" to "toolTip". Gives more info what it represents. Unless it's not a tooltip, still "empty" would be redundant.

Reply:
It's a description (paragraph) that is rendered when CargoAPI is empty. ShowCargo  normally renders "No cargo" but TooltipMakerAPI implementation of showShips and showCargo was inconsistent. I added the checks and render it myself instead.

Reply:
> It's a description (paragraph) that is rendered when CargoAPI is empty.
Ok, I understand it better now. Though having conditions in variable names is not good. I would say "emptyDescription" is fine (with this context I now know).

"emptyCargoDescription" - is another option.